kin-core/src/main/java/kin/core/PaymentWatcher.java

```diff
@@ -61,6 +61,34 @@ public void onEvent(TransactionResponse transactionResponse) {
             });
     }
 
+    /**
+     * Start watching for create account, use {@link #stop()} to stop watching, startCreateAccountListener can be called again after stop.
+     * <p><b>Note:</b> Events will be fired on background thread.</p>
+     *
+     * @param listener listener object for payment events
+     */
+    public void startCreateAccountListener(final WatcherListener<PaymentInfo> listener){
+        checkNotNull(listener, "listener");
+        setStarted();
+        serverSentEvent = server.transactions()
+            .forAccount(accountKeyPair)
+            .cursor(CURSOR_FUTURE_ONLY)
```

Comment:
This should be removed.  Having this cursor implements a time dependency on the caller that should not exist.  The result of watching for `CREATE_ACCOUNT` should be the same, whether or not the account was already created.

Reply:
thanks Avi good point